lua/fzf-lua-frecency/fn_transform.lua

```diff
@@ -42,7 +40,7 @@ M.get_fn_transform = function(opts)
       else
         formatted_score = h.pad_str(h.exact_decimals(score, 2), max_score_len)
       end
-      return ("%s %s"):format(formatted_score, entry)
+      return ("%s%s%s"):format(formatted_score, FzfLua.utils.nbsp, entry)
```

Comment:
Using `FzfLua.utils.nbsp` enables us to skip stripping the score, `FzfLua.path.entry_to_file` will take care of that automatically.